K8s-deployment/Charts/auditing-server/templates/metering/deployment.yaml

```diff
@@ -0,0 +1,173 @@
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
```

Comment:
Add deployment same as verticle name - ``immudb`` as folder name and in values such as -`` Values.immudb.updateStrategy``